editor/animation_track_editor.cpp

```diff
@@ -4989,7 +4987,7 @@ void AnimationTrackEditor::_update_step(double p_new_step) {
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 	undo_redo->create_action(TTR("Change Animation Step"));
-	float step_value = p_new_step;
+	float step_value = Math::round(p_new_step * FACTOR) / FACTOR;
```

Comment:
```suggestion
	// To ensure that the conversion results are consistent between serialization and load, the value is snapped with 0.625 to be a rational number.
	float step_value = Math::round(p_new_step * FACTOR) / FACTOR;
```

Reply:
```suggestion
	float step_value = Math::snap(p_new_step, 0.0625);
```
By the way, is this not enough?

Reply:
Ah, I didn't notice that there was a function for snapping to the nearest value. But yes, that is effectively the value being snapped to currently. Will include those changes along with changing it from a hard coded value to a variable, add a comment saying that the variable should be the reciprocal of a power of 2 (1/2, 1/4, 1/8, etc), and lastly ensure that the FPS max range is clamped to a value less than 10,000 which is entirely reasonable.